src/Bicep.Core.Samples/Files/Dependencies_LF/main.sourcemap.bicep

```diff
@@ -19,6 +19,8 @@ var dependentVar = {
 }
 
 var resourceDependency = {
+//@        "dependencies": {
+//@        }
```

Comment:
Shouldn't these lines be associated with the *use* of `resourceDependency` on line 80 rather than the definition of `resourceDependency` here?

Reply:
My guess is both the LHS & RHS of line 78 are being mapped to the same line in the JSON:
```bicep
dependencies: resourceDependency
```